frontend-react/src/components/MessageTracker/MessageDetails.test.tsx

```diff
@@ -167,27 +168,72 @@ jest.mock("react-router-dom", () => ({
 describe("RSMessageDetail component", () => {
     test("url param (messageId) feeds into network hook", () => {
         mockUseMessageDetails.mockReturnValueOnce({
-            messageDetails: MOCK_EMPTY_MESSAGE_DETAIL as RSMessageDetail,
+            messageDetails: MOCK_EMPTY_MESSAGE_DETAIL,
         });
         render(<MessageDetails />);
         expect(mockUseMessageDetails).toHaveBeenCalledWith(TEST_ID);
     });
 
-    test("renders expected content", async () => {
+    test("renders expected content", () => {
         mockUseMessageDetails.mockReturnValueOnce({
```

Comment:
This is a much larger point of discussion for the entire team so no changes needed at this point, but I think we should move all test setup/teardown work into `beforeEach`/`beforeAll` and `afterEach`/`afterAll` hooks and only/mostly running expectations in the `test` blocks.  The reason for this is that it makes it much easier to isolate expectations based on specific setup conditions and elevate common setup/teardown that would otherwise be redundant.

Again, no changes needed at this point, but I wanted to throw this thought out there!

Reply:
Worth making a ticket for @stephenkao ?

Reply:
@etanb made a ticket [here](https://app.zenhub.com/workspaces/experience-607d9d5e68b95200150fec37/issues/gh/cdcgov/prime-reportstream/8279)